test/fixtures/ts-demo/tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+	"compilerOptions": {
+		"declaration": true
```

Comment:
Any chance we could make this a default?

Reply:
It seems difficult to make it the default in a friendly way.

`rollup-plugin-typescript2` only enables me to force it with `tsconfigOverride` but the user would not be able to disable it (which feels wrong...)

I feel like that if you're writing a typescript module, having a `tsconfig.json` is practically a given (because it enables a lot of features in most editors/IDEs) so it is not a big pain.

Reply:
made an issue in `rollup-plugin-typescript2` to enable setting the default behaviour in the absence of explicit user intent.

Reply:
the feature enabling tsconfig less usage should be coming in the next version of rollup-typescript-plugin2. I feel like we can merge this one (as there is currently no support at all for declaration files) and I'll happily do another PR with the proper defaulting when it becomes available.

Or maybe you prefer to wait a bit @developit ?